components/CollaboratorInput/CollaboratorTypeDropdown.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react'
+import { DropdownList } from './DropdownList' // Import the reusable component
```

Comment:
Please remove dev comments.
```suggestion
import { DropdownList } from './DropdownList'
```